arithmetization/src/main/java/net/consensys/linea/zktracer/module/romlex/RomLex.java

```diff
@@ -188,9 +189,8 @@ public void callRomLex(final MessageFrame frame) {
 
       case CALL, CALLCODE, DELEGATECALL, STATICCALL -> {
         final Address calledAddress = Words.toAddress(frame.getStackItem(1));
-        final Address contractAddress = frame.getContractAddress();
-        final int depNumber = hub.deploymentNumberOf(contractAddress);
-        final boolean depStatus = hub.deploymentStatusOf(contractAddress);
+        final int depNumber = hub.deploymentNumberOf(calledAddress);
+        final boolean depStatus = hub.deploymentStatusOf(calledAddress);
```

Comment:
The `frame.contractAddress` is the address of the bytecode executing in the current execution context (or frame.) What we need is the deployment information of the code we are trying to `EXTCODECOPY` from